docs/book/v1/intro.md

```diff
@@ -44,3 +44,70 @@ the response is created immediately.
 
 If the route is not explicit, *all* request methods are checked to provide a
 list of possible request methods to the client.
+
+## Quick Start
```

Comment:
For consistency with other documentations, create a separate page for the quick start. Insert this page in the navigation after the installation page.